### PR TITLE
Watchdog

### PR DESCRIPTION
**Pull request by cehbrecht**
## Overview

This PR updates emu to use the next pywps (v4.4) with jobqueue. 

## Related Issue / Discussion

https://github.com/geopython/pywps/pull/505

## Additional Information

We should wait with merge until pywps PR is on master.

### CODE REVIEW

**Comment by cehbrecht** [comment]
The cookiecutter template will be updated according to these changes.

**Comment by tlvu** [comment]
Sorry for delay, just back from the holidays.  Will try to get this reviewed by tomorrow.

File: Makefile [hunk 1]
```diff
@@ -25,10 +25,8 @@ help:
 	@echo "  help              to print this help message. (Default)"
 	@echo "  install           to install app by running 'pip install -e .'"
 	@echo "  develop           to install with additional development requirements."
-	@echo "  start             to start $(APP_NAME) service as daemon (background process)."
-	@echo "  stop              to stop $(APP_NAME) service."
```
Comment (tlvu) [inline Makefile]: Is there a good reason to remove `stop` and `status` make targets?

File: Dockerfile [hunk 2]
```diff
@@ -22,10 +22,13 @@ RUN conda env create -n wps -f environment.yml
 # Install WPS
 RUN ["/bin/bash", "-c", "source activate wps && python setup.py develop"]
 
+# Init DB
+RUN ["/bin/bash", "-c", "source activate wps && pywps -c /opt/wps/pywps.cfg migrate"]
```
Comment (tlvu) [inline Dockerfile]: Just curious, why we need this DB migrate/init step now and not before?  Isn't pywps always using a DB (sqlite or postgres) before?

Comment (cehbrecht) [reply Dockerfile]: Before there were no updates to the pywps db schema. With the "watchdog" pywps we will have one and Jachym has added alembic to pywps which needs this migration step.

File: Makefile (hunk 1 above)
Comment (cehbrecht) [reply Makefile]: I can't they there is a good reason ... but currently I don't have the daemon mode for the pywps service implemented in the pywps command line interface. We can bring it back in later steps.

File: Dockerfile (hunk 2 above)
Comment (tlvu) [reply Dockerfile]: Ah, understood, thanks.

File: Makefile (hunk 1 above)
Comment (tlvu) [reply Makefile]: How long is "later steps"?  If it's a while, in order to not forget, could we either add a TODO comment to the code or open an issue?

Comment (cehbrecht) [reply Makefile]: I have opened an issue: #96.

**Comment by tlvu** [comment]
As seen in the pywps PR https://github.com/geopython/pywps/pull/505#issuecomment-572267375 I've found some problems with this emu.  Not sure if it's my side (bad deployment into our PAVICS stack, bad jupyter env) or it's a real bug.

**Comment by cehbrecht** [comment]
will not use this PR.

**Pull request closed without merging**